### PR TITLE
lab: Make the heading of each section `#`

### LINKED ISSUE #118: lab: Make the heading of each section `#` instead of `##`
Docusaurus uses the name of the file instead of the first heading as the title page. This way we end up with sections looking like [this](https://open-education-hub.github.io/operating-systems/Lab/Data/Process%20Memory/content/process-memory) where the title is set to `process-memory` instead of `Process Memory`:

image

The Markdown files for each section have to be updated and the topmost heading changed from `##` to `#`. The rest of the headings probably need to be bumped one level higher as well since Markdownlint doesn't allow adjacent headings to differ by more than one `#`. Look into lab content files as well as quizzes.

### PR DESCRIPTION
**Pull request by teodutu**
Fixes #118.

### CODE REVIEW

**Comment by teodutu** [comment]
The linter errors are caused by the "etc." word. `markdownlint` sees the dot succeeding it as the end of a sentence.

**Comment by Alex-deVis** [comment]
> The linter errors are caused by the "etc." word. `markdownlint` sees the dot succeeding it as the end of a sentence.
This is easy to fix. There is a list of words to be ignored because they contain punctuation. You can find it here: `operating-systems/.github/workflows/rules/md104.js`.

**Comment by teodutu** [comment]
> 

Thanks!

File: content/chapters/software-stack/lab/content/common-functions.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-### Common Functions
+## Common Functions
```
Comment (Alex-deVis) [inline content/chapters/software-stack/lab/content/common-functions.md:1]: You missed this file.

File: content/chapters/software-stack/lab/content/common-functions.md [hunk 2]
```diff
@@ -79,7 +79,7 @@ We see that we have greater printing flexibility with the `printf()` function.
 However, one downside of the current implementation is that it makes a system call for each character.
 This is inefficient and could be improved by printing a whole string.
 
-#### Practice
+### Practice
```
Comment (Alex-deVis) [inline content/chapters/software-stack/lab/content/common-functions.md:82]: Change this too.

**Comment by Alex-deVis** [comment]
I see that your changes pretty much address [MD041](perating-systems/.github/workflows), which was disabled in the config file `operating-systems/.github/workflows/config/config.json`. Reenable it and we can check your changes using the linter.

**Review by Alex-deVis — CHANGES_REQUESTED** [review]
You missed one file. Add the `MD041` and we can merge this based on the linter`s result.

**Review by Alex-deVis — APPROVED** [review]
Nice!

**Pull request merged**